packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -213,17 +213,7 @@ export function useDrag(options: DragOptions): DragResult {
   };
 
   let modality = useDragModality();
-  let message: string;
-  if (!isDragging) {
-    if (modality === 'touch' && !hasDragButton) {
-      message = 'dragDescriptionLongPress';
```

Comment:
were we able to delete a string?

Reply:
No, but now standalone drags don't need long press. That's only needed within a collection.

Reply:
ahhhh, thanks